sys/include/net/sock/udp.h

```diff
@@ -367,6 +367,11 @@ typedef struct {
  * @pre `(sock != NULL)`
  * @pre `(remote == NULL) || (remote->port != 0)`
  *
+ * @warning If you create a socket you are responsible for receiving messages
+ *          sent to it by calling @ref sock_udp_recv.
+ *          If you only want to send without receiving, use @ref sock_udp_send
+ *          instead with @p sock set to NULL.
```

Comment:
Not sure if using `@p sock` is semantically correct here. Usually it refers to a parameter of the function that is documented. Rather use `` `sock` `` here.

```suggestion
 *          instead with `sock` set to NULL.
```